src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -124,7 +124,7 @@ const ReportActionItemSingle = (props) => {
                             {_.map(personArray, (fragment, index) => (
                                 <ReportActionItemFragment
                                     key={`person-${props.action.reportActionID}-${index}`}
-                                    accountID={accountID}
+                                    accountID={String(actorAccountID)}
```

Comment:
Hmm do we _need_ this to be a string? Everywhere else we're trying to force accountIDs to be numbers, so I'm hoping we can just change the expected prop type of `accountID` in ReportActionItemFragment, but maybe I'm missing something